### PR TITLE
Support for Plex, Prowl and growl.

### PR DESCRIPTION
**Pull request by tester22**
This gives support for Plex notifications to update the library when download is done.
Growl & Prowl notifications when a download is started or done.

### CODE REVIEW

**Comment by simonk83** [comment]
Prowl!   I've been hanging for this :D

File: app/lib/growl.py [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+class GrowlRegistrationPacket:
+  """Builds a Growl Network Registration packet.
+     Defaults to emulating the command-line growlnotify utility."""
+
+  def __init__(self, application="NetGrowl", password = None ):
+    self.notifications = []
+    self.defaults = [] # array of indexes into notifications
+    self.application = application.encode("utf-8")
+    self.password = password
+    print password
```
Comment (RuudBurger) [inline app/lib/growl.py:105]: Be sure to remove this.

**Comment by RuudBurger** [comment]
Can't get Growl working on OSX

**Comment by tester22** [comment]
Strange I have it working, you need to enable "Network" and "Listen for incomming messages" like this:
http://emberapp.com/tester22/images/growl

**Comment by RuudBurger** [comment]
Got it! Maybe add the tip: "Be sure to enable notification over the network, in Growl > Network"

**Comment by RuudBurger** [comment]
Nice work btw! Looking awesome :D

**Comment by tester22** [comment]
Will do, and this is how twitter looks in action:
https://twitter.com/#!/tester22/status/54476219504275456

2011/4/3 RuudBurger
reply@reply.github.com:

File: app/lib/growl.py (hunk 1 above)
Comment (tester22) [inline app/lib/growl.py:105]: Forgot to remove that debugging code.

**Comment by tester22** [comment]
Ok will do it later, have a lot of changes in the config/index.html
file to support twitter at the moment.

2011/4/3 RuudBurger
reply@reply.github.com:

**Pull request merged**